docs/guide/persistence_cache.md

```diff
@@ -123,7 +123,9 @@ for information on how to configure Redis.
 
 Out of the box in `app/config/cache_pool/cache.redis.yml` you'll find a default example that can be used.
 
-For eZ Platform Cloud/Platform.sh: This is automatically configured in `app/config/env/platformsh.php` if you have enabled Redis as `rediscache` Platform.sh service.
+!!! cloud "eZ Platform Cloud"
```

Comment:
note: this formating can be used on the same sentence for memcached

Reply:
You're right! I added the same note for the memecached fragment, thanks! :)